net/FlatBuffers/FlatBufferBuilder.cs

```diff
@@ -387,7 +387,7 @@ public VectorOffset EndVector()
         /// Creates a vector of tables.
         /// </summary>
         /// <param name="offsets">Offsets of the tables.</param>
-        public VectorOffset CreateVectorOfTables<T>(Offset<T>[] offsets) where T : struct
+        public VectorOffset CreateVectorOfTables<T>(Offset<T>[] offsets)
```

Comment:
I am worried about these changes to the library, that apply regardless of the 'csharp_class' attribute.